### DISCUSSION TITLE
start python script in new tmux session

### DISCUSSION THREAD

**Discussion by Kolvir73** [tmux-python/libtmux#491] [General]
Is it possible using libtmux to create a new session and then run a script in it from another python script?

Ideally, in a python program,  I'd like to start a new tmux session, rename it, run 'python3 script.py' and then detach it. 
I see how I can start the new session, but am missing how to run 'python3 script.py', which really is just a shell command.
Can it be done with `server.cmd()` somehow? I have experimented with it some without success and will work on it more, but wanted to know if it was even possible.

**Comment by tony** [comment]
Hi and welcome!

Does `Pane.send_keys` work ([source](https://github.com/tmux-python/libtmux/blob/96a09c0d229ad871e08868ab3c0e81c7c4fcca96/src/libtmux/pane.py#L194-L245), [docs](https://libtmux.git-pull.com/reference/panes.html#libtmux.Pane.send_keys))?

```python
pane = session.attached_window.attached_pane

pane.send_keys('python3 ./script.py', enter=True)
```
